package.json

```diff
@@ -66,7 +68,7 @@
     "test": "mocha --opts mocha.opts",
     "pretest": "npm run build",
     "build": "rm -rf dist && rollup -c",
-    "prepare": "npm run build",
+    "prepare": "cd ./node_modules/rollup-dependency-tree && npm install && npm run build",
```

Comment:
I'm assuming this is because it's an unpublished package

Reply:
yeah, I'll remove after publishing the package